TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/claw/Claw.java

```diff
@@ -6,7 +6,7 @@
 public class Claw {
 
     private static float pos;
-    public static final Servo clawServo;
+    public static Servo clawServo;
```

Comment:
Why remove the final?

Reply:
because once there is a final there you cant declare the hardware of this servo which leads to an error
ok?

Reply:
Sure